### PR TITLE
`{:existing_module, module_name, exports}` type

### PR DESCRIPTION
**Pull request by am-kantox**
According to the comment by José, I implemented the ability to check exports as well.

Addresses #103.

### CODE REVIEW

File: lib/nimble_options.ex [hunk 1]
```diff
@@ -927,6 +930,37 @@ defmodule NimbleOptions do
     end
   end
 
+  defp validate_type({:existing_module, module_name, exports}, key, value) do
```
Comment (am-kantox) [inline lib/nimble_options.ex:933]: We might also accept a behaviour here if desired.

**Comment by whatyouhide** [comment]
I'm starting to have doubts too here. In general, this is kind of mimicking the "behaviour" functionality, where what you might want to be checking is that the given module implements a given behaviour. The fact that the module must be loaded seems more like application logic than option validation?

**Comment by am-kantox** [comment]
Accepting a behaviour as a third parameter is not an issue, also we have `BehaviourModule.behaviour_info(:callbacks)` out of the box.

The check for the module availability in compile time might be a duty of options validation in some cases (static consistency check,) or not, and in the latter case we still would have bare `:atom` type. 

I am fine doing it through a custom checker, though.

**Comment by whatyouhide** [comment]
Closed as per my comment in #103. Thanks so much anyways @am-kantox! 💟

**Pull request closed without merging**